### PR TITLE
Docs: update phpDocs

### PR DESCRIPTION
**Pull request by ramiy**

### CODE REVIEW

File: core/dynamic-tags/manager.php [hunk 1]
```diff
@@ -33,8 +33,20 @@ public function __construct() {
...
 	 * @access public
+	 *
+	 * @param string   $text           Dynamic tag text.
+	 * @param array    $settings       The dynamic tag settings.
+	 * @param callable $parse_callback The functions that renders the dynamic tag.
+	 *
+	 * @return string|string[]|mixed A string or an array of strings with the return
+	 *                               value from each tag callback function.
 	 */
 	public function parse_tags_text( $text, array $settings, callable $parse_callback ) {
```
Comment (bainternet) [inline core/dynamic-tags/manager.php]: @KingYes callable is form php 5.4 only

File: core/settings/base/manager.php [hunk 2]
```diff
@@ -10,7 +10,7 @@
 }
 
 /**
- * Elementor settings base manager class.
+ * Elementor settings base manager.
```
Comment (bainternet) [inline core/settings/base/manager.php]: base?

File: core/settings/base/model.php [hunk 3]
```diff
@@ -8,7 +8,7 @@
 }
 
 /**
- * Elementor base settings model class.
+ * Elementor base settings model.
```
Comment (bainternet) [inline core/settings/base/model.php]: base???????????????

File: includes/base/element-base.php [hunk 4]
```diff
@@ -6,7 +6,7 @@
 }
 
 /**
- * Element Base.
+ * Elementor base element.
```
Comment (bainternet) [inline includes/base/element-base.php]: element base

File: includes/base/skin-base.php [hunk 5]
```diff
@@ -6,7 +6,7 @@
 }
 
 /**
- * Skin Base.
+ * Elementor base skin.
```
Comment (bainternet) [inline includes/base/skin-base.php]: skin base

File: includes/base/widget-base.php [hunk 6]
```diff
@@ -6,7 +6,7 @@
 }
 
 /**
- * Widget Base.
+ * Elementor base widget.
```
Comment (bainternet) [inline includes/base/widget-base.php]: widget base

File: includes/controls/base-data.php [hunk 7]
```diff
@@ -77,8 +77,18 @@ public function get_value( $control, $settings ) {
 	}
 
 	/**
+	 * Parse dynamic tags.
+	 *
+	 * Goes through all the controls and renders all the dynamic tags.
```
Comment (bainternet) [inline includes/controls/base-data.php]: Goes => iterates

File: includes/controls/base-multiple.php [hunk 8]
```diff
@@ -8,9 +8,9 @@
 /**
  * Elementor base multiple control.
```
Comment (bainternet) [inline includes/controls/base-multiple.php]: control base multiple

File: includes/controls/base-ui.php [hunk 9]
```diff
@@ -8,7 +8,7 @@
 /**
  * Elementor base UI control.
```
Comment (bainternet) [inline includes/controls/base-ui.php]: Base_UI_Control

File: includes/controls/base-units.php [hunk 10]
```diff
@@ -8,7 +8,7 @@
 /**
  * Elementor base units control.
```
Comment (bainternet) [inline includes/controls/base-units.php]: Control_Base_Units

File: includes/controls/groups/base.php [hunk 11]
```diff
@@ -8,7 +8,7 @@
 /**
  * Elementor base group control.
```
Comment (bainternet) [inline includes/controls/groups/base.php]: Group_Control_Base

File: includes/schemes/base.php [hunk 12]
```diff
@@ -6,10 +6,10 @@
 }
 
 /**
- * Scheme base class.
+ * Elementor base scheme.
```
Comment (bainternet) [inline includes/schemes/base.php]: Scheme_Base

File: includes/template-library/sources/local.php [hunk 13]
```diff
@@ -17,7 +17,7 @@
 }
 
 /**
- * Elementor template library local source class.
+ * Elementor template library local source.
```
Comment (bainternet) [inline includes/template-library/sources/local.php]: Source_Local

File: includes/template-library/sources/remote.php [hunk 14]
```diff
@@ -8,7 +8,7 @@
 }
 
 /**
- * Elementor template library remote source class.
+ * Elementor template library remote source.
```
Comment (bainternet) [inline includes/template-library/sources/remote.php]: Source_Remote

File: modules/history/revisions-manager.php [hunk 15]
```diff
@@ -11,6 +11,14 @@
 	exit; // Exit if accessed directly.
 }
 
+/**
+ * Elementor revisions manager.
```
Comment (bainternet) [inline modules/history/revisions-manager.php]: Elementor history revisions manager.

File: modules/library/documents/library-document.php [hunk 16]
```diff
@@ -7,14 +7,28 @@
...
+ * Elementor library document.
+ *
+ * Elementor library document handler class is responsible for handling
+ * a document of the library type.
+ *
+ * @since 2.0.0
+ */
 abstract class Library_Document extends Document {
 
 	const TAXONOMY_TYPE_SLUG = 'elementor_library_type';
```
Comment (bainternet) [inline modules/library/documents/library-document.php]: describe const

File: modules/library/documents/library-document.php [hunk 17]
```diff
@@ -7,14 +7,28 @@
 	exit; // Exit if accessed directly
 }
 
+/**
+ * Elementor library document.
```
Comment (bainternet) [inline modules/library/documents/library-document.php]: elementor library documents library document??

File: modules/library/documents/page.php [hunk 18]
```diff
@@ -7,12 +7,26 @@
 	exit; // Exit if accessed directly
 }
 
+/**
+ * Elementor page document.
```
Comment (bainternet) [inline modules/library/documents/page.php]: elementor library documents page

File: modules/library/documents/section.php [hunk 19]
```diff
@@ -5,12 +5,26 @@
 	exit; // Exit if accessed directly
 }
 
+/**
+ * Elementor section document.
```
Comment (bainternet) [inline modules/library/documents/section.php]: elementor library documents section

File: core/settings/base/manager.php (hunk 2 above)
Comment (ramiy) [reply core/settings/base/manager.php]: from the namespace.

File: modules/library/documents/page.php (hunk 18 above)
Comment (ramiy) [reply modules/library/documents/page.php]: Elementor page library document.

File: modules/library/documents/library-document.php (hunk 17 above)
Comment (ramiy) [reply modules/library/documents/library-document.php]: Elementor section library document.

File: modules/library/documents/section.php (hunk 19 above)
Comment (ramiy) [reply modules/library/documents/section.php]: Elementor section library document.

File: core/dynamic-tags/manager.php [hunk 20]
```diff
@@ -33,8 +33,20 @@ public function __construct() {
 	}
 
 	/**
+	 * Parse text with multiple dynamic tags.
+	 *
+	 * Receives a text with a multiple dynamic tags, runs the callback functions
+	 * for each tag and returns the value from the callback.
```
[the remark this answers is not in the document]
Comment (ramiy) [reply core/dynamic-tags/manager.php]: I've rewrote the description.

File: core/settings/base/model.php (hunk 3 above)
Comment (ramiy) [reply core/settings/base/model.php]: fixed

File: includes/base/element-base.php (hunk 4 above)
Comment (ramiy) [reply includes/base/element-base.php]: fixed

File: includes/base/skin-base.php (hunk 5 above)
Comment (ramiy) [reply includes/base/skin-base.php]: fixed

File: includes/base/widget-base.php (hunk 6 above)
Comment (ramiy) [reply includes/base/widget-base.php]: fixed

File: includes/controls/base-data.php (hunk 7 above)
Comment (ramiy) [reply includes/controls/base-data.php]: fixed

File: includes/controls/base-multiple.php (hunk 8 above)
Comment (ramiy) [reply includes/controls/base-multiple.php]: fixed

File: includes/controls/base-ui.php (hunk 9 above)
Comment (ramiy) [reply includes/controls/base-ui.php]: Implementing the class name convention.

File: includes/controls/base-units.php (hunk 10 above)
Comment (ramiy) [reply includes/controls/base-units.php]: fixed

File: includes/controls/groups/base.php (hunk 11 above)
Comment (ramiy) [reply includes/controls/groups/base.php]: fixed

File: includes/schemes/base.php (hunk 12 above)
Comment (ramiy) [reply includes/schemes/base.php]: fixed

File: modules/history/revisions-manager.php (hunk 15 above)
Comment (ramiy) [reply modules/history/revisions-manager.php]: fixed

File: modules/library/documents/library-document.php (hunk 17 above)
Comment (ramiy) [reply modules/library/documents/library-document.php]: really??

**Pull request merged**